File: niuniu-take-out-springboot3/server/src/main/java/common/niuniu/service/UserService.java
package common.niuniu.service;

import common.niuniu.dto.UserDTO;
import common.niuniu.dto.UserLoginDTO;
import common.niuniu.entity.User;

public interface UserService {
    User wxLogin(UserLoginDTO userLoginDTO);

    User getUser(Integer id);

    void update(UserDTO userDTO);
}

File: niuniu-take-out-kotlin/server/src/main/kotlin/common/niuniu/service/UserService.kt
package common.niuniu.service

import common.niuniu.dto.UserDTO
import common.niuniu.dto.UserLoginDTO
import common.niuniu.po.User

interface UserService {
    fun wxLogin(userLoginDTO: UserLoginDTO?): User?

    fun getUser(id: Int?): User?

    fun update(userDTO: UserDTO?)
}
